universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts

```diff
@@ -18,5 +18,7 @@ export const createPreconfiguredServices = async (provider: providers.Provider,
   services.sdk.priceObserver.getCurrentPrices = () => {
     return Promise.resolve({ETH: {USD: 1, DAI: 1, ETH: 1}});
   };
+  await services.sdk.tokensDetailsStore.fetchTokensDetails();
+  await services.sdk.fetchRelayerConfig();
```

Comment:
Move to sdk.start()

Reply:
We can use `sdk.start()` here, but we need to remember about:
 - remove `sdk.start()` and `sdk.tokensDetailsStore.fetchTokensDetails()` (lines 31 and 32) [connect.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/ui/connect.tsx)
- remove `sdk.tokensDetailsStore.fetchTokensDetails()` (line 26) [CreateAccount.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/ui/CreateAccount.tsx)
- add `sdk.stop()` in after in [CreateAccount.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/ui/CreateAccount.tsx)
- remove `sdk.tokensDetailsStore.fetchTokensDetails()` (line 16) [setupUI.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/helpers/setupUI.tsx)

Reply:
We can add two functions to sdk: fetchData, startObserver. Then change sdk.start() to 
```js
start() {
    fetchData();
    startObserver();
}
```

Then in the test, we will use only fetchData where it is necessary.

Reply:
@marekkirejczyk